### ISSUE TITLE
Flag tabindex=-1 in aria-hidden for review unless there is a modal open

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3486]
From [Element with aria-hidden has no focusable content](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/6cfa84/proposed/#failed-example-5)

```html
<div aria-hidden="true">
	<button tabindex="-1">Some button</button>
</div>
```

I'm reluctant to outright fail these, but there are cases where this can be a problem, so I think axe-core should call these out as incomplete, unless there is a modal open on the page.
